base/Dockerfile.nf.webconsole

```diff
@@ -0,0 +1,31 @@
+#
+# Dockerfile responsible to compile webconsole NF from sources on the host
+#
+
+FROM free5gc/base as my-base
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Get Free5GC
+COPY free5gc/ $GOPATH/src/free5gc/
+
+RUN cd $GOPATH/src/free5gc \
+    && make webconsole
+
+# Alpine is used for debug purpose. You can use scratch for a smaller footprint.
+FROM alpine:3.15
+
+WORKDIR /free5gc
+RUN mkdir -p config/TLS/ public
+
+# Copy executables
+COPY --from=my-base /go/src/free5gc/webconsole/bin/webconsole ./webui

```

Comment:
webconsole binary is not being located in generic path like other free5gc NFs. Thus for now we create another Dockerfile.